gpbft/src/lib.rs

```diff
@@ -141,9 +150,9 @@ mod tests {
         let step = Phase::Commit;
         let supplemental_data = SupplementalData {
             commitments: keccak_hash::H256::zero(),
-            power_table: vec![],
+            power_table: Cid::default(),
```

Comment:
Would a possible lack of power table be better represented by `Option`?

Reply:
Default CID means that there's no power table, correct? The default CID still has a set multihash and version and, subjectively, checking if something is not of certain value to assert it's not there is bug-prone. If it's an `Option` then the developer knows that they need to check whether it exists and not deduce that they need to check against a particular hard-coded CID.

Not a terribly strong feeling, but I'd advise to reconsider the approach. It's similar to an `Address` - it can be defaulted to something, but what is the meaning of it? It's the reason that it's hidden in FVM under the `testing` feature. It's similar thing to a checksum of something - having a checksum of 00000 would mean (to me) that you actually ran the digest method and you somehow obtained all zeroes therefore you should quit your job and buy yourself an island from the billions you made on crypto. :)

Reply:
In this particular case this is used for convenience. `Cid::default()` is as good for this test as `Cid::blahblahblah`. 
Cid is not nullable in Go code, so it should not be in ours I reckon.

Reply:
If we look through the codebase - it's also not being used for anything rather than comparison to another cid that's manually hashed, so there is no reason to change the approach.